cmd/machine-api-operator/main.go

```diff
@@ -9,7 +9,7 @@ import (
 
 const (
 	componentName      = "machine-api-operator"
-	componentNamespace = "kube-system"
+	componentNamespace = "openshift-machine-api-operator"
```

Comment:
do we need to have componentNamespace and default namespace?

Reply:
Keep in mind the installer is currently using `openshift-cluster-api` for our stuff.

Reply:
@bison This should be transparent so the operator would be deployed to "openshift-machine-api-operator" and the cluster-api stack along with the machineSet object will fall under "openshift-cluster-api" as the installer config is overriding the targetNamespace, @vikaschoudhary16 correct?

Reply:
right. If targetNamespace is not set in the configmap, only then it will be initialized to openshift-machine-api-operator.